### PR TITLE
test: removing unnecessary optionals and increasing branch cov - 2

### PR DESCRIPTION
**Pull request by suyashpatil78**
### <samp>🤖 Generated by Copilot at 0f2ef84</samp>

This pull request enhances the unit testing and code style of the `setupFormInit()` function in the `add-edit-expense-5.spec.ts` file. It removes unnecessary commas, adds missing semicolons, and aligns the code indentation. It also adds new test cases to check the form initialization for different expense types and modes.

### <samp>🤖 Generated by Copilot at 0f2ef84</samp>

> _Sing, O Muse, of the skillful coder who refined_
> _The `setupFormInit()` function with a clear mind,_
> _Removing commas that cluttered the code like weeds,_
> _And adding tests that covered all possible deeds._

### <samp>🤖 Generated by Copilot at 0f2ef84</samp>

*  Remove extra commas from various function calls for code style consistency ([link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639L190-R194), [link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639L273-R273), [link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639L298-R298), [link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639L323-R323), [link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639L348-R348), [link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639L647-R647))
*  Add test cases for `setupFormInit()` function in `add-edit-expense-5.spec.ts` to increase code coverage and ensure functionality ([link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639R872-R992), [link](https://github.com/fylein/fyle-mobile-app/pull/2504/files?diff=unified&w=0#diff-ba304f41b9ceb84ec3edd25c39133e53d238dc01ad0cb7b04fbd999aa2300639R1229-R1274))

## Clickup
app.clickup.com

## Code Coverage
Please add code coverage here

## UI Preview
Please add screenshots for UI changes

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts [hunk 1]
```diff
@@ -869,6 +869,127 @@ export function TestCases5(getTestBed) {
         expect(component.setCategoryOnValueChange).toHaveBeenCalledTimes(1);
       }));
 
+      it('should setup form and set payment mode as default payment mode if selectedPaymentMode is undefined', fakeAsync(() => {
```
Comment (suyashpatil78) [inline src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts:872]: change here

File: src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts [hunk 2]
```diff
@@ -1105,6 +1226,52 @@ export function TestCases5(getTestBed) {
         });
         expect(component.setCategoryOnValueChange).toHaveBeenCalledTimes(1);
       }));
+
```
Comment (suyashpatil78) [inline src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts:1229]: change here

File: src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts (hunk 1 above)
Comment (OmkarJ13) [reply src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts:872]: Can we make this test smaller by moving common parts together? This is too big right now

File: src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts (hunk 2 above)
Comment (OmkarJ13) [reply src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts:1229]: Same as above

**Review by OmkarJ13 — COMMENTED** [review]
Added comments

File: src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts (hunk 1 above)
Comment (suyashpatil78) [reply src/app/fyle/add-edit-expense/add-edit-expense-5.spec.ts:872]: Will add a TODO as discussed @OmkarJ13

**Review by shishirams222 — APPROVED** [review]
Please get green from Omkar for large tests

**Pull request merged**